modules/geo-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -86,6 +111,26 @@ export default class _MeshLayer extends SimpleMeshLayer {
     });
     return materialParser;
   }
+
+  calculateObjectIdsPickingColors(attribute) {
+    const {objectIds} = this.props;
+
+    if (!objectIds) {
+      return;
```

Comment:
If you set `attribute.constant = true; attribute.value = [0,0,1];` here you won't need a uniform flag

Reply:
I completely remove this checking because it is already checked earlier on the [line 39](https://github.com/visgl/deck.gl/pull/5757/files#diff-b1bc28dc9141b4f38fee3377dc1103f06ae14b01afcea7593c50c817b5cebe05R39)